valohai/internals/yaml.py

```diff
@@ -70,19 +69,30 @@ def parse_input(key: str, value: Any) -> Input:
         value["name"] = key
         return Input.parse(value)
 
+    # Check and ignore the local path file
+    if isinstance(value, List):
+        checked_value = [val for val in value if not is_local_file_path(val)]
+    else:
+        checked_value = "" if is_local_file_path(value) else value
+
     has_wildcards = any(
-        "*" in uri for uri in ([value] if isinstance(value, str) else value)
+        "*" in uri
+        for uri in (
+            [checked_value] if isinstance(checked_value, str) else checked_value
+        )
     )
     empty_default = (
-        not value
-        or isinstance(value, List)
-        and (len(value) == 0 or len(value) == 1 and not value[0])
+        not checked_value
+        or isinstance(checked_value, List)
+        and (
+            len(checked_value) == 0 or len(checked_value) == 1 and not checked_value[0]
+        )
     )
```

Comment:
I think this (72..90) would be greatly simplified if you run `value` through `listify()` (from `valohai_yaml.utils`; it's used around this codebase too) first, so you'd always be dealing with a list of strings :)

Reply:
Yeah!

You could keep track of whether the original value was a string, though, and return it as such (i.e. "unpeel" the 1-item list right at the end) if you want to be neat about it.